doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1045,56 +1045,62 @@ A subscription is represented by a JSON object with the following fields:
 * `id`: Subscription unique identifier. Automatically created at creation time.
 * `subject` : It s an object that describes the subject of the subscription.
 * `notification` : It is an object that describes the notification received by the subscriber.
-* `options` : It is a dictionary which can contain different parameters related to the subscription, including:
-  * `duration`: Duration of the subscription in ISO8601 format. Infinite if not specified.
+* `duration`: Duration of the subscription in ISO8601 format. Infinite if not specified.
 
-A 'subject' contains the following subfields:
+A `subject` contains the following subfields:
 
-* `entity` : Id or pattern id which identifies the affected entities (optional)
+* `id` : Id of the affected entities (optional). It cannot be use at the same time than `idPattern`.
```

Comment:
Typo: same time *as*

And yes, we should somehow say that id|idPattern is mandatory, but just one at a time ...

Perhaps using (semi-mandatory*) and then the longer explanation "in the footer"?